Slider/Assets/Scenes/Village.unity

```diff
@@ -139278,7 +139618,13 @@ PrefabInstance:
       objectReference: {fileID: 0}
     - target: {fileID: 3652435011426292518, guid: eb8cd11432469439eb214403b87b5b92, type: 3}
       propertyPath: conds.Array.data[0].dialogueChain.Array.data[6].dialogue
-      value: This seems awfully concerning to me...
+      value: Or maybe my senses are getting mixed up with what i saw on the tele
```

Comment:
Do TV's exist in slider? we never see any (unless you count jungle) but there's computers so presumably so?